src/base/constants.py

```diff
@@ -44,3 +44,16 @@ def _IsDevAppServer():
 DEBUG = IS_DEV_APPSERVER
 
 TEMPLATE_DIR = os.path.sep.join([os.path.dirname(__file__), '..', 'templates'])
+
+AUTHORIZATION_METHOD = 'basic_auth'
```

Comment:
No! You don't change the source code to configure your program's runtime parameters. You don't keep config hardcoded in the repo. You don't keep secrets like passwords in git.